packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.module.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import 'ovh-api-services';
+
+import routing from './cold-archive.routing';
+import service from './cold-archive.service';
+import component from './cold-archive.component';
+
+const moduleName = 'ovhManagerPciStoragesColdArchive';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
+    'oui',
+    'ovh-api-services',
```

Comment:
**suggestion:** Can we avoid using `ovh-api-services` in new projects an use `$http` instead. It makes the refactoring easy when we move from angular.js?

Reply:
Yep sure i will remove it.
I don't use it anymore from my side, just a bad copy-past.

Thanks to have highlight it  👍